tools/tcpconnlat.py

```diff
@@ -84,8 +84,8 @@ def positive_float(val):
     // XXX: switch some to u32's when supported
     u64 ts_us;
     u64 pid;
-    u64 saddr;
-    u64 daddr;
+    u32 saddr;
+    u32 daddr;
     u64 ip;
     u64 dport;
```

Comment:
could you help convert pid to u32 and dport to u16 as well? Once this is done, you can remove the comments.

The same for below ipv6_data_t data structure.